Revit_Core_Engine/Create/Solid.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Autodesk.Revit.DB;
+using BH.oM.Geometry;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Create solid object from filled region limited by bottom and top elevation.")]
+        [Input("filledRegion", "Filled region that is extruded to the solid.")]
+        [Input("bottomElevation", "Bottom elevation of the solid.")]
+        [Input("topElevation", "Top elevation of the solid.")]
+        [Output("solid", "Solid representation of extruded filled region.")]
+        public static Solid Solid(this FilledRegion filledRegion, double bottomElevation, double topElevation)
```

Comment:
I think the general approach is not to use `this` keyword in `Create` methods - these methods should remind factory pattern.
```suggestion
        public static Solid Solid(FilledRegion filledRegion, double bottomElevation, double topElevation)
```